### PR TITLE
Skip this test when running on OpenShift OCP 4.1

### PR DESCRIPTION
**Pull request by kevinearls**
Signed-off-by: Kevin Earls <kearls@redhat.com>

### CODE REVIEW

File: test/e2e/examples2_test.go [hunk 1]
```diff
@@ -53,6 +55,13 @@ func (suite *ExamplesTestSuite2) TestSimplestExample() {
 }
 
 func (suite *ExamplesTestSuite2) TestWithBadgerExample() {
+	if isOpenShift(t) {
+		ocpVersion := os.Getenv("OCP_VERSION")
+		if len(ocpVersion) > 0 && strings.HasPrefix(ocpVersion, "4.1") {
```
Comment (pavolloffay) [inline test/e2e/examples2_test.go]: Can you explain why it is skipped only on OCP 4.1?

Comment (kevinearls) [reply test/e2e/examples2_test.go]: At the moment it's only failing on 4.1 and we need to decide how much time we want to invest in investigating this, as we probably won't be supporting OCP 4.3 for much longer

Comment (pavolloffay) [reply test/e2e/examples2_test.go]: so It was passing before on 4.1 but not it is failing?

**Comment by kevinearls** [comment]
I've opened issue #870 to track this problem.

File: test/e2e/examples2_test.go (hunk 1 above)
Comment (kevinearls) [reply test/e2e/examples2_test.go]: Yes, that's correct.

**Comment by kevinearls** [comment]
No longer applicable

**Pull request closed without merging**